src/vidpf.rs

```diff
@@ -0,0 +1,735 @@
+// Copyright (c) 2024 Cloudflare, Inc.
```

Comment:
We're not the copyright holders, ISRG is.
```suggestion
```

Reply:
there are no contribution guidelines for this project, btw.